src/blog/understanding-ecmascript-part-1.md

```diff
@@ -130,7 +130,7 @@ That is, we inspect a Completion Record; if it’s an abrupt completion, we retu
 > 2. `ReturnIfAbrupt(obj)`
 > 3. `Bar(obj)`. (If we’re still here, `obj` is the value extracted from the Completion Record.)
 
-And now [the question mark](https://tc39.es/ecma262/#sec-returnifabrupt-shorthands) comes into play: `? Foo()` is equivalent to `ReturnIfAbrupt(Foo())`.
+And now [the question mark](https://tc39.es/ecma262/#sec-returnifabrupt-shorthands) comes into play: `? Foo()` is equivalent to `ReturnIfAbrupt(Foo())`. Using a shorthand is practical: we don’t need to write the error handling code explicitly each time.
```

Comment:
I don't feel this explains it as well as Shu's comment you posted on Twitter. Could you add some modified version of that comment here for better clarity?